### PR TITLE
userguide: add installation from Ubuntu PPA section - v1

### PR DESCRIPTION
**Pull request by jufajardini**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/6231

Describe changes:
- Bring more info about installing suri from Ubuntu PPAs from redmine to our documentation
- fix a reference link about DPDK in the installation guide that wasn't truly a link
- add a commit message example in our contribution guidelines, to make it even easier to understand what our style is like.

### CODE REVIEW

File: doc/userguide/install.rst [hunk 1]
```diff
@@ -170,18 +170,85 @@ Follow these steps from your Suricata directory::
 Binary packages
 ---------------
 
-Ubuntu
-^^^^^^
+Ubuntu from Personal Package Archives (PPA)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (jufajardini) [inline doc/userguide/install.rst:174]: Wasn't sure where this should go, but this seemed like the best place, from what already exists.

File: doc/userguide/install.rst [hunk 2]
```diff
@@ -170,18 +170,85 @@ Follow these steps from your Suricata directory::
...
     sudo add-apt-repository ppa:oisf/suricata-stable
     sudo apt-get update
+
+Then, you can install the latest stable with::
+
     sudo apt-get install suricata
 
+If you want Suricata with built-in (enabled) debugging::
+
+    sudo apt-get install suricata-dbg
```
Comment (jasonish) [inline doc/userguide/install.rst:191]: I wonder if this should be moved down along with beta releases.

File: doc/userguide/install.rst [hunk 3]
```diff
@@ -170,18 +170,85 @@ Follow these steps from your Suricata directory::
...
+    sudo apt-get upgrade suricata
+
+Remove
+""""""
+
+To remove Suricata from your system::
+
+    sudo apt-get remove suricata
+
+Getting Beta or RC Releases
```
Comment (jasonish) [inline doc/userguide/install.rst:212]: I'd spell out `RC`, at least the first time.  Or simply use `Pre-release Versions`.

**Comment by jufajardini** [comment]
Feedback incorporate in: https://github.com/OISF/suricata/pull/9332

**Pull request closed without merging**